boards/shields/rpi_pico_lcd/doc/index.rst

```diff
@@ -758,6 +840,88 @@ sample: :zephyr:code-sample:`zephyr:input-dump`.
          I: input event: dev=gpio_keys        SYN type= 1 code= 28 value=1
          I: input event: dev=gpio_keys        SYN type= 1 code= 28 value=0
 
+   .. group-tab:: Waveshare Pico LCD 1.44
+
+      Print the input events related to the five on-shield joystick keys
```

Comment:
**praise (blocking):** fix description, there is no joystick on the Waveshare Pico LCD 1.44 – check / verify also all other new shield descriptions

Reply:
done